CHANGELOG.md

```diff
@@ -133,6 +133,12 @@ New modules
   Function.Identity.Instances
   ```
 
+* Algerbaic properties:
```

Comment:
typo

Reply:
I think errors/conflicts in `CHANGELOG` are moot now. I'd be working from a fresh copy form the current `master` whenever that day comes ;-)